phpunit.xml

```diff
@@ -1,4 +1,4 @@
-<phpunit bootstrap="./App/Tests/phpunit.php">
+<phpunit bootstrap="./App/TESTS/phpunit.php">
```

Comment:
Changing the name of the folders will renders all test links in the final report unusable